glean-core/src/ping/mod.rs

```diff
@@ -195,6 +195,12 @@ impl PingMaker {
         Ok(pings_dir)
     }
 
+    fn get_tmp_dir(&self, data_path: &Path) -> std::io::Result<PathBuf> {
```

Comment:
Would you kindly add documentation for this function? IMHO it's also worth explaining why we're not simply saving files to /tmp